lib/generateClass.js

```diff
@@ -205,6 +213,66 @@ function writeFromJson(def, cw) {
   cw.ln('return inst;');
 }
 
+/**
+ * Generates a JSON serializer for the element
+ * @param {DataElement} def - The definition of the SHR element to generate a serializer for
+ * @param {CodeWriter} cw - The CodeWriter instance to use during generation
+ * @private
+ */
+function writeToJson(def, cw) {
+  // If the element is an Entry, put those fields on the JSON object first
+  if (def.isEntry) {
+    cw.ln(`const inst = this._entryInfo.toJSON();`);
+    const url = `http://standardhealthrecord.org/spec/${def.identifier.namespace.replace('.', '/')}/${className(def.identifier.name)}`;
+    cw.ln(`inst['shr.base.EntryType'] = { 'Value' : '${url}' };`);
+  } else if (def.identifier.name !== 'EntryType') {
+    const url = `http://standardhealthrecord.org/spec/${def.identifier.namespace.replace('.', '/')}/${className(def.identifier.name)}`;
```

Comment:
NBD, but this `url` definition is repeated from above.  You could pull it out and define it once before the `if` statement.